docs/TodoList.md

```diff
@@ -52,93 +52,20 @@ myapp
 Creating a Dispatcher
 ---------------------
 
-Now we are ready to create a dispatcher. Here is an naive example of a Dispatcher class, written with JavaScript promises, polyfilled with Jake Archibald's [ES6-Promises](https://github.com/jakearchibald/ES6-Promises) module.
+Now we are ready to create a dispatcher. We provide you with a default implementation, `Dispatcher`:
```

Comment:
It might be worth adding mentioning npm, Bower and the repo here, like we do here: http://facebook.github.io/flux/docs/overview.html#a-single-dispatcher (generated from https://github.com/facebook/flux/blame/master/docs/Overview.md#L66).